packages/cloudflare/package.json

```diff
@@ -45,14 +45,14 @@
     "tiny-glob": "^0.2.9"
   },
   "peerDependencies": {
-    "astro": "^4.2.0"
+    "astro": "^4.3.5"
```

Comment:
I believe a `minor` bump to the `peerDependency` range should also correspond with a `minor` bump to the package, but I don't see `@astrojs/cloudflare` included in the changesets.